docs/technical/services-api.md

```diff
@@ -58,3 +58,16 @@ Commands for service bots and commands for the main PyLink bot have two main dif
 ### Featured commands
 
 Commands for service bots can also be marked as *featured*, which shows it with its command arguments in the default `LIST` command. To mark a command as featured, use `myservice.add_cmd(cmdfunc, 'cmdname', featured=True)`.
+
+### Config Sections
+
+All service bots have the following available configuration directives.
+
+**Note**: It is recommended to have a service named in the same way the config section is defined.
+This being '`pylink::` should be registered as `utils.registerService("pylink", desc=desc)`'.
+
+- `nick`, `ident` - Sets the nick and ident of the service bot, respectively.
+- `joinmodes` - Sets the mode that the service will ask for on join. (if the mode isn't supported its ignored)
```

Comment:
"Sets the mode**s**" (plural). Combinations like "v", "o", "ao", etc. should all work provided the IRCd supports it.